relay-server/src/services/store.rs

```diff
@@ -1253,7 +1298,7 @@ struct SpanKafkaMessage<'a> {
         rename = "_metrics_summary",
         skip_serializing_if = "Option::is_none"
     )]
-    metrics_summary: Option<BTreeMap<Cow<'a, str>, Option<SpanMetricsSummaries>>>,
+    metrics_summary: Option<BTreeMap<Cow<'a, str>, Option<SpanMetricsSummaries<'a>>>>,
```

Comment:
AFAICT this `Cow` will always become `Cow::Owned`, so we might as well remove the  `#[serde(borrow)]` and use `String`.

Playground link: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=63202f57b8a55bea32004f85b905d235

Reply:
I think I still need the `borrow` because `'a` needs to outlive `'de` for the `tags: <&'a RawValue>`.